public/managehashes.php

```diff
@@ -44,23 +46,62 @@
 <?php RenderToolbar($user, $permissions); ?>
 <script>
   function UpdateHashDetails(user, hash) {
+    $('#warning').html('Status: updating...');
     var name = $.trim($('#HASH_' + hash + '_Name').val());
     var labels = $.trim($('#HASH_' + hash + '_Labels').val());
     var posting = $.post('/request/game/modify.php', { u: user, g: <?php echo $gameID ?>, f: 4, v: hash, n: name, l: labels });
-    posting.done(onUpdateComplete);
+    posting.done(function onUpdateComplete(data) {
+        if (data !== 'OK') {
+            $('#warning').html('Status: Errors...' + data);
+        } else {
+            // Get comment date
+            var date = new Date();
+            var dateStr = date.getUTCDate() + ' ' + shortMonths[date.getUTCMonth()] + ' ' +  date.getUTCFullYear() + '<br>' + date.getUTCHours() + ':' + ('0' + date.getUTCMinutes()).slice(-2);
+
+            // Place comment on correct row depending on if the user has submitted a comment already
+            if ($('#commentTextarea').length) {
+                $('#feed tr:last').before('<tr class="feed_comment localuser system"><td class="smalldate">' + dateStr + '</td><td class="iconscommentsingle"></td><td class="commenttext">' + hash + ' updated by ' + user + '. Description: "' + name + '". Label: "' + labels + '"</td></tr>');
+            } else {
+                $('#feed').append('<tr class="feed_comment localuser system"><td class="smalldate">' + dateStr + '</td><td class="iconscommentsingle"></td><td class="commenttext">' + hash + ' updated by ' + user + '. Description: "' + name + '". Label: "' + labels + '"</td></tr>');
+            }
```

Comment:
```suggestion
        if (data !== 'OK') {
            $('#warning').html('Status: Errors...' + data);
            return;
        }
        // Get comment date
        var date = new Date();
        var dateStr = date.getUTCDate() + ' ' + shortMonths[date.getUTCMonth()] + ' ' +  date.getUTCFullYear() + '<br>' + date.getUTCHours() + ':' + ('0' + date.getUTCMinutes()).slice(-2);

        // Place comment on correct row depending on if the user has submitted a comment already
        if ($('#commentTextarea').length) {
            $('#feed tr:last').before('<tr class="feed_comment localuser system"><td class="smalldate">' + dateStr + '</td><td class="iconscommentsingle"></td><td class="commenttext">' + hash + ' updated by ' + user + '. Description: "' + name + '". Label: "' + labels + '"</td></tr>');
        } else {
            $('#feed').append('<tr class="feed_comment localuser system"><td class="smalldate">' + dateStr + '</td><td class="iconscommentsingle"></td><td class="commenttext">' + hash + ' updated by ' + user + '. Description: "' + name + '". Label: "' + labels + '"</td></tr>');
        }
```
[Code calisthenics - avoid else, return early.](https://freek.dev/2212-avoid-using-else)

Reply:
This suggestion should not be applied via GitHub's offered actions as it will mess with nesting - couldn't add a suggestion that includes deleted lines below.

Reply:
Fixed.